src/main/java/com/github/dockerjava/api/command/DockerCmd.java

```diff
@@ -1,7 +1,8 @@
 package com.github.dockerjava.api.command;
 
-import java.io.Closeable;
+public interface DockerCmd<RES_T> extends AutoCloseable {
```

Comment:
could you describe why Closeable need to be changed to AutoClosable when
`public interface Closeable extends AutoCloseable {` ?